src/components/settings/UserNavigation.tsx

```diff
@@ -0,0 +1,35 @@
+import { useLocation } from 'react-router-dom';
+
+import { cn } from '@/lib';
+
+const TABS = [
+    { name: 'Settings', href: '/settings/user-settings' },
```

Comment:
Can this just be the base `settings` slug?

Reply:
Si